spring-core/src/main/java/org/springframework/core/serializer/support/DeserializingConverter.java

```diff
@@ -70,8 +70,7 @@ public DeserializingConverter(Deserializer<Object> deserializer) {
 
 	@Override
 	public Object convert(byte[] source) {
-		ByteArrayInputStream byteStream = new ByteArrayInputStream(source);
-		try {
+		try(var byteStream = new ByteArrayInputStream(source);) {
```

Comment:
There is no technical reason to close a `ByteArrayInputStream` since its `close()` method is a no-op.

Therefore, please revert this change consistently within this PR.

Reply:
Reverted